invenio_config_tugraz/templates/invenio_app_rdm/help/search.de.html

```diff
@@ -0,0 +1,257 @@
+        >Syntax des regulären Ausdrucks</a
+      >
+      .
+    </p>
+    <h3>Fehlende Werte</h3>
+    <p>
+      Es ist möglich, mit den Feldnamen <code>_exists_</code> und
+      <code>_missing_</code> nach Datensätzen zu suchen,
+      in denen entweder ein Wert fehlt oder die einen Wert in einem bestimmten Feld haben.
+    </p>
+    <p>
+      <strong>Beispiel:</strong>
+      <a href="/search?page=1&amp;size=20&amp;q=_missing_:metadata.additional_titles"
+        ><code>_missing_:metadata.additional_titles</code></a
+      >
+      (alle Datensätze ohne metadata.additional_titles)
+    </p>
+    <p>
+      <strong>Beispiel:</strong>
+      <a href="/search?page=1&amp;size=20&amp;q=_exists_:metadata.creators"
+        ><code>_exists_:metadata.creators</code></a
+      >
+      (alle Datensätze mit metadata.creators)
+    </p>
+    <h3>Erweiterte Konzepte</h3>
+    <h4>Boosten</h4>
+    <p>
+      Sie können den Boost-Operator <code>^</code> verwenden, wenn ein Begriff relevanter ist als ein
+      anderer. Sie können z. B. nach allen Datensätzen suchen, die den Ausdruck
+      <em>open science</em> entweder <em>im title</em> oder im
+      <em>description</em> feld enthalten, aber Datensätze mit dem Ausdruck im
+      <em>title</em> feld höher einstufen:
+    </p>
+    <p>
+      <strong>Beispiel:</strong>
+      <a
+        href="/search?page=1&amp;size=20&amp;q=metadata.title:%22open%20science%22%5E5%20metadata.description:%22open%20science%22"
+        ><code>metadata.title:"open science"^5 metadata.description:"open science"</code></a
+      >
+    </p>
+    <h4>Unschärfe</h4>
+    <p>
+      Mit dem Fuzzy-Operator <code>~</code>können Sie nach
+      Begriffen suchen, die Ihrem Suchbegriff ähnlich, aber nicht genau gleich sind.
+    </p>
+    <p>
+      <strong>Beispiel:</strong>
+      <a href="/search?page=1&amp;size=20&amp;q=oepn~"><code>oepn~</code></a>
+    </p>
+    <p>
+      Die Ergebnisse entsprechen Datensätzen mit ähnlichen Begriffen wie <code>oepn</code> die z. B.
+      auch auf <code>open</code> passen würden.
+    </p>
+    <h4>Näherungssuche</h4>
+    <p>
+      Eine Phrasensuche wie <code>"open science"</code> erwartet standardmäßig alle Begriffe in
+      genau der gleichen Reihenfolge und würde daher z. B. nicht auf einen Datensatz
+      passen, der die Phrase <em>"open access and science"</em> enthält.
+      Eine Proximity-Suche erlaubt, dass die Begriffe nicht in der exakten Reihenfolge stehen
+      und auch andere Begriffe dazwischen enthalten können.
+      Der Grad der Flexibilität wird anschließend durch eine Ganzzahl angegeben:
+    </p>
+    <p>
+      <strong>Beispiel:</strong>
+      <a href="/search?page=1&amp;size=20&amp;q=%22open%20science%22~5"
+        ><code>"open science"~5</code></a
+      >
+    </p>
+    <h4>Wildcards</h4>
+    <p>
+      Sie können Wildcards in Suchbegriffen verwenden, um ein einzelnes Zeichen (mit
```

Comment:
This sentence is super hard to understand.
- 'Operator<code>?</code> operator'
- what means 'null' (either one character or null character)?
- ' Zeichen  (mit' two blanks between 'n' and '('.